attributes/default.rb

```diff
@@ -258,7 +258,7 @@
 
 default['hops']['schema_dir']               = "#{node['hops']['root_url']}/hops-schemas"
 
-default['hops']['ndb']['version']           = "21.04.15"
+default['hops']['ndb']['version']           = "22.10.1"
```

Comment:
I guess the version should be snapshot since it is not a closed version yet 
that is the jar we have https://repo.hops.works/master/ndb-dal-3.2.0.12-SNAPSHOT-22.10.1-SNAPSHOT.jar
```suggestion
default['hops']['ndb']['version']           = "22.10.1-SNAPSHOT"
```